packages/embed-grid/CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+This guide will serve as a reference for contributing to the Deephaven.
+
+## Getting the source
+
+Deephaven uses the [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow).  In this workflow, the [deephaven/deephaven-core](https://github.com/deephaven/deephaven-core) repository contains a minimum number of branches, and development work happens in user-forked repositories.
```

Comment:
There's a few `deephaven-core` references in this file that should be `web-client-ui`.

Reply:
I also moved CONTRIBUTING.md to the root instead of just within these two packages, since it is about contributing to the repo